modules/get-started/pages/whats-new.adoc

```diff
@@ -6,6 +6,41 @@ This topic includes new content added from version 4.29.0 onwards.
 
 For a full list of product updates, and details of updates prior to version 4.29.0, see the https://github.com/redpanda-data/connect/releases[Redpanda Connect release notes^]. For new Cloud-specific features, see xref:redpanda-cloud:get-started:whats-new-cloud.adoc[].
 
+== Version 4.39.0
+
+Highlights of this update include the beta release of three new components for Self-Managed and Cloud, including a new `snowflake_streaming` output. This release also includes additions to the `redpanda` family of components and introduces new features for Self-Managed.
+
+=== Components for Cloud and Self-Managed (beta)
+
+* xref:components:outputs/snowflake_streaming.adoc[`snowflake_streaming` output]
+* xref:components:inputs/timeplus.adoc[`timeplus` input]
+* xref:components:inputs/spicedb_watch.adoc[`spicedb_watch` input]
+
+=== Additional `redpanda` components (beta)
+
+* `redpanda` (xref:components:inputs/redpanda.adoc[input] and xref:components:inputs/redpanda.adoc[output]): Consume message data from or send message data to Kafka brokers.
+* `redpanda_common` (xref:components:inputs/redpanda_common.adoc[input] and xref:components:outputs/redpanda_common.adoc[output]): Consume message data from or send message data to a Redpanda Kafka broker, using credentials from a common xref:components:redpanda/about.adoc[`redpanda` configuration block] (Self-Managed only).
+
+=== Features in Self-Managed
+
+You can now:
+
+* Create an xref:configuration:allow_and_deny_lists.adoc[allow list or deny list] for a Redpanda Connect instance (Self-Managed only)
```

Comment:
Seems like the (Self-Managed only) here is redundant given that this appears under the heading "Self-Managed", no?

Reply:
Removed.